cdhweb/static_src/global/components/person-hero.scss

```diff
@@ -0,0 +1,188 @@
+/* 
+Person hero. Similar to standard hero. Key differences:
+- image ratio (TODO – maybe?)
+- text content – person hero has more complex needs
+*/
+
+// has `.grid-standard`
+.person-hero {
+  --title-top-space: 40px;
+
+  margin-block-end: 56px;
+
+  // h = heading/intro: name, job title, other metadata
+  // c = contact info + social links
+
+  // Layout if there is _no_ image:
+  grid-template-areas:
+    'h h h h h h h h h h h h'
+    'c c c c c c c c c c c c';
+
+  @include sm {
+    margin-block-end: 120px;
+  }
+
+  @include md {
+    --title-top-space: 64px;
+
+    grid-template-areas:
+      'h h h h h h h h h h h h'
+      '. c c c c c c c c c c c';
+  }
+  @include lg {
+    --title-top-space: 80px;
+
+    grid-template-areas:
+      '. h h h h h h h h h h .'
+      '. .  c c c c c c c c .';
+  }
+  @include xl {
+    --title-top-space: 104px;
+  }
+  @include xxl {
+    --title-top-space: 160px;
+    grid-template-areas:
+      '. h h h h h h h h h . .'
+      '. . c c c c c c c c . .';
+  }
+
+  // Different layout if the hero has an image:
+  &:has(img) {
+    grid-template-areas:
+      'i i i i i i i i i i i i'
+      'h h h h h h h h h h h h'
+      'c c c c c c c c c c c c';
+
+    @include md {
+      grid-template-areas:
+        'h h h h h . i i i i i i'
+        '. c c c c c i i i i i i';
+    }
+    @include lg {
+      grid-template-areas:
+        '. h h h h h i i i i i i'
+        '. . c c c c i i i i i i';
+    }
+  }
+}
+
+.person-hero__intro {
+  grid-area: h;
+  margin: var(--title-top-space) 0 0;
```

Comment:
Will do the former.

Reply:
Wait I misread the comment, I was thinking of another issue.

Yeah, OK, so this was because in the _standard_ hero template, the element being selected is the h1, which has its own margin-bottom going on, so it was to override that. However in this template I'm applying the margin to the element _containing_ the h1, and other text that aligns with the h1. I'll move the margin to the h1.

Ugh, ...except that that's not right. I'd not-thinkingly assumed that the h1, like other typographic elements, had a margin bottom. But I deliberately left that out of the rich-text styles, and the reset.scss is setting its margin to `0`.

So yeah, I'll make some adjustments.